arborist/user.go

```diff
@@ -200,18 +227,18 @@ func revokeUserPolicyAll(db *sqlx.DB, username string) *ErrorResponse {
 	return nil
 }
 
-func addUserToGroup(db *sqlx.DB, username string, groupName string) *ErrorResponse {
+func addUserToGroup(db *sqlx.DB, username string, groupName string, expiresAt *time.Time) *ErrorResponse {
 	if groupName == AnonymousGroup || groupName == LoggedInGroup {
 		return newErrorResponse("can't add users to built-in groups", 400, nil)
 	}
 	if username == "" {
 		return newErrorResponse("missing `username` argument", 400, nil)
 	}
 	stmt := `
-		INSERT INTO usr_grp(usr_id, grp_id)
-		VALUES ((SELECT id FROM usr WHERE name = $1), (SELECT id FROM grp WHERE name = $2))
+		INSERT INTO usr_grp(usr_id, grp_id, expires_at)
+		VALUES ((SELECT id FROM usr WHERE name = $1), (SELECT id FROM grp WHERE name = $2), $3)
 	`
```

Comment:
adding `ON CONFLICT (usr_id, grp_id) DO UPDATE SET expires_at = EXCLUDED.expires_at` should be safe here?